wallet/wallet-cli-lib/tests/cli_test_framework.rs

```diff
@@ -298,7 +298,22 @@ impl CliTestFramework {
         let manager_task = manager.main_in_task();
 
         let wallet_options = WalletCliArgs {
-            network: Network::Regtest,
+            network: Network::Regtest(Box::new(config::regtest_options::ChainConfigOptions {
+                chain_magic_bytes: None,
```

Comment:
looks like a good reason to add `ChainConfigOptions::new`

Reply:
I think this is fine... with this huge number of parameters, using positional arguments would be a disaster.

Reply:
I meant `new()` or default

Reply:
can be dropped as discussed